Create a patch in buildroot tslib package to add a button "Quit" in the program ts_test

Signed-off-by: Jérémie Scheer <jeremie.scheer@armadeus.com>

Index: buildroot/package/tslib/tslib-add_quit_button_to_test.patch
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ buildroot/package/tslib/tslib-add_quit_button_to_test.patch	2010-07-07 17:23:42.000000000 +0200
@@ -0,0 +1,45 @@
+--- tslib-1.0/tests/ts_test.c.orig	2010-07-07 17:02:50.000000000 +0200
++++ tslib-1.0/tests/ts_test.c		2010-07-07 16:27:44.000000000 +0200
+@@ -44,7 +44,7 @@ static int button_palette [6] =
+ 	1, 5, 0
+ };
+ 
+-#define NR_BUTTONS 2
++#define NR_BUTTONS 3
+ static struct ts_button buttons [NR_BUTTONS];
+ 
+ static void sig(int sig)
+@@ -152,13 +152,15 @@ int main()
+ 
+ 	/* Initialize buttons */
+ 	memset (&buttons, 0, sizeof (buttons));
+-	buttons [0].w = buttons [1].w = xres / 4;
+-	buttons [0].h = buttons [1].h = 20;
+-	buttons [0].x = xres / 4 - buttons [0].w / 2;
+-	buttons [1].x = (3 * xres) / 4 - buttons [0].w / 2;
+-	buttons [0].y = buttons [1].y = 10;
++	buttons [0].w = buttons [1].w = buttons[2].w = xres / 4;
++	buttons [0].h = buttons [1].h = buttons[2].h = 20;
++	buttons [0].x = xres / 4 - 2 * (buttons [0].w / 3);
++	buttons [1].x = xres / 2 - buttons [0].w / 2;
++	buttons [2].x = 3 * (xres / 4) - buttons [0].w / 3;
++	buttons [0].y = buttons [1].y = buttons [2].y = 10;
+ 	buttons [0].text = "Drag";
+ 	buttons [1].text = "Draw";
++	buttons [2].text = "Quit";
+ 
+ 	refresh_screen ();
+ 
+@@ -196,6 +198,12 @@ int main()
+ 					mode = 1;
+ 					refresh_screen ();
+ 					break;
++				case 2:
++					close_framebuffer();
++					fflush(stderr);
++					fflush(stdout);
++					exit(1);
++					break;
+ 				}
+ 
+ 		printf("%ld.%06ld: %6d %6d %6d\n", samp.tv.tv_sec, samp.tv.tv_usec,
